terraform/deployments/govuk-publishing-platform/outputs.tf

```diff
@@ -17,6 +17,19 @@ output "signon_security_groups" {
   description = "Used by ECS RunTask to run short-lived tasks with the same SG permissions as the signon ECS Service."
 }
 
+output "content-store" {
+  value = {
+    draft = {
+      task_definition_cli_input_json = module.draft_content_store.cli_input_json,
+      security_groups                = module.draft_content_store.security_groups,
```

Comment:
by tying both live and draft together, you assume that one draft can't be updated independently from live/vice versa?

Reply:
Yeah, this is by design, on the assumption that we won't ever be in the situation where we want to deploy draft independently from live, or web independently from worker.

I _think_ that's more or less the situation with Jenkins at the moment, although I haven't checked.

Reply:
Yeah, I had a look at what Jenkins does.

There's just one "Deploy_App" job, which specifies the app (say "content-store"). You don't get to specify draft or live.

Capistrano has a [`set_servers`](https://github.com/alphagov/govuk-app-deployment/blob/master/recipes/set_servers.rb) rake task, which uses `ovuk_node_list -c content-store` to get a list of servers to deploy to. That will include draft and live versions.

So people should already be used to the idea that it's not possible to deploy draft and live independently. Personally, I think this constraint is a good thing - I can't think of a legitimate reason to have them on different versions.

Reply:
I was more thinking if we want to tweak CPU/memory of live instances because of increased traffic, we may not need to do that for the draft ones. Anyway, we can cross that bridge when we get there if needed.

Reply:
Ah, they do get different task definitions, so they don't have to have the same memory / CPU. They just have to be deployed at the same time.